index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio Website</title> 
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <header class="header">
+    <nav class="nav">
+        <a href="#"> Welcome</a>
+        <a href="#menu"><img src="./assets/icons/hamburger.svg" alt=""></a>    
+    </nav>
+  </header>
+  <section class="headline">
+      <h1 class="headline-title">Hey there. I’m Solomon. <br><span>I’m a software developer</span> </h1>
+      <p class="headline-description">I can help you build a product ,feature or website Look through
+        some of my  work and experience! If you like what you see and have 
+        a project you need coded, don’t hestiate to contact me.
+      </p>
+      <ul class="headline-icons">
+        <li><a href="#twitter" ><img src="./assets/icons/twitter.svg" alt="Twitter"></a></li>
+        <li><a href="#linkedIn" ><img src="./assets/icons/Linkedin_icon.svg" alt="LinkedIn"></a></li>
+        <li><a href="#medium" ><img src="./assets/icons/medium.svg" alt="Medium"></a></li>
+        <li><a href="#github" ><img src="./assets/icons/github.svg" alt="Github"></a></li>
+        <li><a href="#angellist" ><img src="./assets/icons/angellist.svg" alt="Angellist"></a></li>
+      </ul>
+      <div class="headline-dropdown">
+        <a href=""><img src="/assets/icons/Disabled.svg" alt="Dropdown"></a>
```

Comment:
[OPTIONAL] Kindly change the image's path to `"./assets/icons/Disabled.svg"` to fix the image displayed. Adding the dot means that the path starts from the current file's directory.